internal/controller/state/conditions/conditions.go

```diff
@@ -354,6 +370,16 @@ func NewRouteUnsupportedValue(msg string) Condition {
 	}
 }
 
+// NewRouteUnsupportedField returns a Condition that indicates that the Route includes an unsupported field.
+func NewRouteUnsupportedField(msg string) Condition {
+	return Condition{
+		Type:    string(RouteConditionUnsupportedField),
+		Status:  metav1.ConditionTrue,
+		Reason:  string(RouteReasonUnsupportedField),
+		Message: fmt.Sprintf("Some parameters are ignored due to an error: %s", msg),
```

Comment:
It's not really an error


```suggestion
		Message: fmt.Sprintf("Some parameters are ignored because they are not supported: %s", msg)
```